debian/control

```diff
@@ -0,0 +1,26 @@
+Source: litecli
+Section: python
+Priority: optional
+Maintainer: LiteCLI Contributors <info@litecli.com>
+Uploaders: Joshua M. Boniface <joshua@boniface.me>
```

Comment:
These two lines need to be set to the correct people. I used the dummy `info@litecli.com` for the maintainer and myself as the uploader but technically `Uploaders:` is only needed for "real" Debian packages and could be removed.

Reply:
I actually just removed Uploaders: but Maintainer: should be set to the right person/people (usually, the release manager).